### DISCUSSION TITLE
Samsung browser defaulting to Scene-Viewer

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4659] [Q&A] [answered]
I'm testing on a Samsung S9 and all model-viewer content is opened in Scene-Viewer instead of WebXR (my pages and model-viewer examples). Chrome works as expected in all cases.
I have also tested on Samsung Tab S4 and it works there as expected.

This WebXR content works in Samsung browser as expected https://immersive-web.github.io/webxr-samples/immersive-ar-session.html

Samsung S9 - Android 10
Samsung Internet v23.0.1.1

**Comment by elalish** [comment]
I think that means Samsung doesn't support WebXR on the S9. Check with them.

**Reply by danilokrulj** [reply]
But the link I posted works on S9 (its WebXR), so it can't be that WebXR doesn't work on S9.

**Reply by elalish** [reply]
Oh, sorry - I missed that line. Can you view the JS console? It should print some messages about why it had to fall back to Scene Viewer.

**Reply by danilokrulj** [reply]
![image](https://github.com/google/model-viewer/assets/92090684/e0a351bd-e923-4777-9116-96fa49c46839)

These are the errors I get in the console.

**Reply by elalish** [reply]
If Chrome is working as expected, then this is definitely looking like a Samsung WebXR bug of some kind. Do they have a place to file issues?

**Reply by danilokrulj** [reply]
I'm not really sure if they have a place for issues.

**Reply by danilokrulj** [reply]
Its definitely a Samsung Browser issue, I just realized that the browser wasn't updated on the S4 Tab, as soon as I updated the browser it also started defaulting to Scene-viewer. I found out where to report the issue on their side.

https://forum.developer.samsung.com/t/samsung-browser-not-supporting-webxr-on-s9/29672

**Reply by danilokrulj** [reply, marked as answer]
UPDATE:
I updated the Samsung Internet to the newest version available (24.0.3.4) and the issue is now resolved.
